src/tasks/tasks.py

```diff
@@ -139,7 +140,7 @@ def count_examples(self, splits=['train', 'val', 'test']):
 
     def tokenizer_is_supported(self, tokenizer_name):
         ''' Check if the tokenizer is supported for this task. '''
-        return tokenizer_name == utils.TOKENIZER.__class__.__name__
+        return True
```

Comment:
Just delete this everywhere if all instances return `True`?

Reply:
:+1: